src/utils/deprecate.js

```diff
@@ -0,0 +1,32 @@
+// Mark that a method should not be used.
+// Returns a modified function which warns once by default.
+// If --no-deprecation is set, then it is a no-op.
+exports.deprecate = function deprecate(fn, msg) {
```

Comment:
Let's use es6 syntax here since we already have babel configured and ready to go.

Reply:
Fixed, thanks.